# from legacy, might need to be retuned
ready_pose:
  kicking_team: [[-0.93, 0.0, 0.0],
                 [-0.66, 0.4, 0.0],
                 [-0.66, -0.4, 0.0],
                 [-0.6, 0.0, 0.0],
                 [-0.5, -0.27, 0.0],
                 [-0.25, 0.0, 0.0],
                 [-0.44, -0.3, 0.0]]
  non_kicking_team: [[-0.9, 0.0, 0.0],
                     [-0.7, 0.45, 0.0],
                     [-0.7, -0.45, 0.0],
                     [-0.6, 0.0, 0.0],
                     [-0.1, -0.0, 0.0],
                     [-0.3, 0.3, 0.0],
                     [-0.15, -0.6, 0.0]]

desired_distances:
  des_distance_goal: 0.6  # y_pos_left_goal = 0.8
  to_ball: 0.25 # for engge and penalty kick
  observation_ball: 1.0
  penalty_kick_goalpost: 0.3

target_pose_optimization_parameters:
  # gradient descent
  scale_gradient: 5.0
  step_size_gradient: 0.5 # [m]

  # cover cost
  cover_scale: 5.0
  cover_variance: 1.0

  # ball CoM cost
  ball_CoM_scale: 100.0

  # avoidance cost
  avoidance_scale_opponent: 200.0
  avoidance_variance_opponent: 0.5
  avoidance_scale_team: 100.0
  avoidance_variance_team: 0.5

  # goal free cost
  goal_free_scale: 500.0

  # offense defense cost
  offense_defense_scale: 1700.0
  offense_defense_variance: 0.8
  offense_defense_x_switch_behavior: 0.0
